pkg/cluster/cluster.go

```diff
@@ -760,6 +763,30 @@ func (c *Cluster) initHumanUsers() error {
 	return nil
 }
 
+func (c *Cluster) initHumanUsers() error {
+
+	for _, postgresSuperuserTeam := range c.OpConfig.PostgresSuperuserTeams {
+		err := c.initTeamMembers(postgresSuperuserTeam, true /*isPostgresSuperuserTeam*/)
```

Comment:
I would say if we need this commentary, it's better to incorporate this information into the argument itself - e.g. replace boolean with enum.

Reply:
this is just  a convenience commentary, imo not worth an enum for just two cases